packages/lodestar-fork-choice/src/forkChoice/forkChoice.ts

```diff
@@ -528,6 +528,13 @@ export class ForkChoice implements IForkChoice {
     return this.protoArray.iterateNodes(toHexString(blockRoot)).map(toBlockSummary);
   }
 
+  /**
+   * The same to iterateBlockSummaries but this gets non-ancestor nodes instead of ancestor nodes.
+   */
+  public unIterateBlockSummaries(blockRoot: Root): IBlockSummary[] {
```

Comment:
any better names?

Reply:
How about `iterateNonAncestors`

Reply:
@wemeetagain I changed the name accordingly